templates/e-commerce/data/build.js

```diff
@@ -0,0 +1,107 @@
+const fs = require('fs');
```

Comment:
The products will definitely be static.

This script was built to help generate a product library quickly, we are going to update the product images, prices etc as time goes on. So I could omit this file and keep it as a public gist for others to use if they need to? What do you think?

There is no need for the products to get generated everytime we run a gulp build.